prd.ini

```diff
@@ -0,0 +1,13 @@
+[db]
```

Comment:
Looks like this is your inventory file.  Do we want to have this under version control?